src/audio/Kconfig

```diff
@@ -76,3 +76,31 @@ config COMP_TEST_KEYPHRASE
 	  Provides basic functionality for use in testing of keyphrase detection pipelines.
 
 endmenu
+
+menu "Data formats"
+
+config FORMAT_S16LE
+	bool "Support S16LE"
+	default y
+	help
+	  Support 16 bit processing data format with sign and in little endian format 
```

Comment:
It's also worth mentioing here that saying NO here can save memory foot[rint at the cost of restricting supported topologies. This can be done incremetally though.

Reply:
Can you fix the CI checkpatch incrementally too, it's just asking for more comments on th enew configs.

Reply:
Previous checkpatch wanted also long descriptions. FYI for checkpatch long enough paragraph - 4 lines, so imo  it shouldn't be even warn just CHECK, but it's like this in kernel repo ...